app/_src/introduction/concepts.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Concepts
+---
+
+In this page we will introduce concepts that are core to understanding Kuma.
+
+## Control Plane
+
+The control-plane is the central management layer of {{ site.product_name }}. It is responsible for configuring and managing the behavior of the data plane,
+which handles the actual traffic between services.
+
+## Data-Plane Proxy / Sidecar
```

Comment:
so is `Data-Plane Proxy` official way now of spelling it now? We have multiple kind of it thought of it.

We could use this concepts page as glossary for how to properly name stuff thorough our docs

Reply:
also in policy introduction we use a lot just `proxy` should we include it in list: Data-Plane Proxy / Sidecar / Proxy?

Reply:
We should add "control-plane" and "control-plane" to Vale to disallow them

Reply:
100% I never remember what we should do so I'd love something in Vale.

So what is it? Data plane? Control Plane? or Data-plane and Control-plane?